### ISSUE TITLE
Add default paths as R drive locations for data

### ISSUE THREAD

**Issue by edwardsmarc** [NCC-CNC/wtw-data-prep#1]
Eventually, this repo will be accessed by staff on the AVD. The file paths to common datasets could then be set to the R drive path by default (e.g. for the NCC grid and pre-prepped 1km national datasets).

Users accessing the repo locally would just change the paths to match the data locations on their systems.

**Comment by DanWismer** [comment]
I am wondering if we could have a config file that will set the paths programmatically if we are on local vs AVD?

**Comment by edwardsmarc** [comment]
This could work well. We could source the config in `initiate_project.R`.

My only reservation is that hiding the file paths in a config might make it harder for R beginners to troubleshoot the scripts.

**Comment by DanWismer** [comment]
True, we could just store them in a variable in the script then. We can figure out where the script is sitting. If its on the R: drive, use the R:drive path. If not, use a local path .. something like that.

**Comment by edwardsmarc** [comment]
Placing this on hold until we find out whether GIS staff can use AVD for data prep, or if they need to run it locally which is the current recommendation.
